scripts/git/prune_old_tags.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
```

Comment:
I recommend adding `set -euo pipefail` unless there's a specific reason you don't want that

(random reference from the interwebz: https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425)

Reply:
FYI you can write it all on one line like I have in the comment